manager/testdata/unittests/basetaxonomy/catalog.structs.schema.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
How come this is part of this PR?  No work on taxonomies was done as part of this PR.  Right?  Same comment for all the taxonomy files

Reply:
It looks like a mistake I'll remove them. @froesef committed a pr to ignore them: https://github.com/fybrik/fybrik/pull/800